introspect_test.go

```diff
@@ -32,58 +34,60 @@ func TestConnsAndStreamIntrospect(t *testing.T) {
 	require.NoError(t, err)
 	_, err = s2.Write([]byte(msg2))
 	require.NoError(t, err)
-	// wait for the metres to kick in
+
+	// wait for the metrics to kick in
 	for {
-		cis, err := swarms[0].IntrospectConnections(introspect.ConnectionQueryParams{Output: introspect.QueryOutputFull})
+		cis, err := swarms[0].IntrospectConnections(introspection.ConnectionQueryParams{Output: introspection.QueryOutputFull})
 		require.NoError(t, err)
+
 		if cis[0].Traffic.TrafficOut.CumBytes != 0 {
 			break
```

Comment:
@raulk You might want to replace this with `require.Eventually`

Reply:
@aarshkshah1992 yeah, feel free to submit a follow-up PR. I'm gonna go ahead and merge all of this now.